store/cache/cache.go

```diff
@@ -148,6 +148,14 @@ func (c *cache) Keys(ctx context.Context, tableName string, pattern string) ([]s
 	return c.Client.Keys(ctx, encodeToCacheKey(tableName, pattern)).Result()
 }
 
+func (c *cache) Scan(ctx context.Context, tableName string, cursor uint64, count int64, pattern string) ([]string, uint64) {
+	if count > config.DefaultConfig.Cache.MaxScan {
```

Comment:
If there are more keys than the maxScan limit, what happens then?

Reply:
maxScan just controls how many keys goes in a batched stream response. this is to prevent server sending a big bulk to client in one go.